src/main.rs

```diff
@@ -46,7 +68,7 @@ async fn main() -> Result<()> {
             let tableoforders = get_orders_list(
```

Comment:
in order to have a rusty syntax that can make it easier to read the code for new devs, let's use snake case for variables, in this case:

`let tableoforders` => `let table_of_orders`